### DISCUSSION TITLE
error stat metadata no such file or directory

### DISCUSSION THREAD

**Discussion by turtle01** [Altinity/clickhouse-backup#565] [Q&A] [answered]
Moving to a new server
Copied the backup to the new server in the folder /var/library/clickhouse/backup/
clickhouse-list of backups - sees it

but when restoring, I get an error
error status /var/lib/clickhouse/backup/2022-10-05T07-15-55/ metadata: there is no such file or directory

**Comment by Slach** [comment]
when you run 
`clickhouse-backup list local`
do you see backup size?

are you sure you copied backup completelly?
which use do you use when run `clickhouse-backup restore` ?

could you share
`ls -la /var/lib/clickhouse/backup/2022-10-05T07-15-55/`
on new server?

**Comment by turtle01** [comment]
-- clickhouse-backup list local
2022-10-05T07-15-55   ???   17/11/2022 16:04:33   local

--  clickhouse-backup restore ?
clickhouse-backup restore "2022-10-05T07-15-55"
2022/11/17 20:42:07.161246  info clickhouse connection prepared: tcp://localhost:9000 run ping logger=clickhouse
2022/11/17 20:42:07.164039  info clickhouse connection open: tcp://localhost:9000 logger=clickhouse
2022/11/17 20:42:07.164300  info SELECT value FROM `system`.`build_options` where name='VERSION_INTEGER' logger=clickhouse
2022/11/17 20:42:07.171433  info SELECT * FROM system.disks; logger=clickhouse
2022/11/17 20:42:07.173544  info clickhouse connection closed logger=clickhouse
2022/11/17 20:42:07.173670 error stat /var/lib/clickhouse/backup/2022-10-05T07-15-55/metadata: no such file or directory

-- ls -la /var/lib/clickhouse/backup/2022-10-05T07-15-55/
total 24
drwxr-xr-x 3 root root  4096 Nov 17 16:04 .
drwxr-x--- 3 root root  4096 Nov 17 16:04 ..
-rw-r--r-- 1 root root 12017 Oct  5 10:15 parts.hash
drwxr-xr-x 5 root root  4096 Nov 17 16:04 shadow

**Comment by Slach** [comment]
Sorry, but this is not backup created with clickhouse-backup
We don't create `parts.hash` file during execute `clickhouse-backup create`

How exactly did you create your backup?

**Comment by Slach** [comment]
> 2022-10-05T07-15-55 ??? 17/11/2022 16:04:33 local

this mean clickhouse-backup can't recognize `<backup_name>/metadata.json`

**Comment by turtle01** [comment, marked as answer]
apparently in a hurry I took something wrong from the server
But the old server is no longer there (
it's good that these are just logs that are no longer interesting.
I have already tried to create a backup on a new server, everything is completely different there)
Thank you for your responsiveness

**Reply by ilyas424** [reply]
did you solve the problem? did everything work out?
